src/validation.cpp

```diff
@@ -5138,3 +5134,15 @@ void ChainstateManager::MaybeRebalanceCaches()
         }
     }
 }
+
+ChainstateManager::~ChainstateManager() {
```

Comment:
nit in fc13e841aa2937dfeb067a36f699c7343947cd6d: According to clang-format missing newline?

Reply:
Done 607d73a8a21bb3b3848b45cbb9844d58e9934c42